tools/serverpod_cli/lib/src/analyzer/models/entity_dependency_resolver.dart

```diff
@@ -1,11 +1,10 @@
 import 'dart:math';
 
-import 'package:super_string/super_string.dart';
-
 import 'package:serverpod_cli/src/analyzer/models/checker/analyze_checker.dart';
 import 'package:serverpod_cli/src/analyzer/models/definitions.dart';
 import 'package:serverpod_cli/src/generator/types.dart';
 import 'package:serverpod_shared/serverpod_shared.dart';
+import 'package:super_string/super_string.dart';
```

Comment:
I hope this was not by some convention, as my current VS Code sorted the imports this way.